### PR TITLE
chore(deps): update django to 3.2.19

### PR DESCRIPTION
**Pull request by trowik**

### CODE REVIEW

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Change looks good.

I think the commit message could be a bit better. This commit updates all the dependencies and pins django to `^3.2.19`.

**Review by open-dynaMIX — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -45,6 +45,7 @@ python-redmine = "^2.3.0"
 sentry-sdk = "^1.9.5"
 whitenoise = "^6.2.0"
 django-hurricane = "^1.3.4"
+openpyxl = "3.0.10"
```
Comment (fugal-dy) [inline pyproject.toml:48]: add some explanation and, if you have it, the link to an issue or something so we know when to unpin

Comment (trowik) [reply pyproject.toml:49]: Found the core issue, I guess. Added a comment and an explanation in the commit.

Comment (open-dynaMIX) [reply pyproject.toml:49]: Might also make sense to link the issue as done [here](https://github.com/adfinis/mySAGW/blob/main/api/requirements-base.txt#L15).

Comment (trowik) [reply pyproject.toml:49]: ah nice, there's already an issue for that.
Thanks a lot

**Pull request merged**